Source/WebCore/Modules/identity/IdentityCredentialsContainer.cpp

```diff
@@ -50,8 +51,20 @@ void IdentityCredentialsContainer::get(CredentialRequestOptions&& options, Crede
     if (!performCommonChecks(options, promise))
         return;
 
+    if (!options.digital || options.publicKey) {
+        promise.reject(Exception { ExceptionCode::NotSupportedError, "Only digital member is supported."_s });
+        return;
+    }
+
+    // FIXME: waiting on conclusion to https://github.com/WICG/digital-credentials/issues/147
+    // if (options.mediation != MediationRequirement::Required) {
+    //     promise.reject(Exception { ExceptionCode::NotAllowedError, "Meditation is required."_s });
+    //     return;
+    // }
```

Comment:
Let's make this a tracked FIXME instead and remove the commented code.

The bugzilla will have a space for the code we should commit based on the resolution of WICG/digital-credentials#147.

```suggestion
    // FIXME: <https://webkit.org/b/bug-number> waiting on conclusion to https://github.com/WICG/digital-credentials/issues/147
```